openedx_events/event_bus/tests/test_loader.py

```diff
@@ -77,20 +77,20 @@ def test_missing_attribute(self):
                 "component will be inactive"
         ]):
             loaded = _try_load(
-                setting_name="EB_LOAD_PATH",
+                setting_name="EB_LOAD_PATH", args=(1), kwargs={'2': 3},
                 expected_class=dict, default={'def': 'ault'},
             )
         assert loaded == {'def': 'ault'}
 
-    @override_settings(EB_LOAD_PATH='builtins.len')
+    @override_settings(EB_LOAD_PATH='builtins.dict')
     def test_bad_args_for_callable(self):
         with assert_warnings([
                 "Failed to load <class 'dict'> from setting EB_LOAD_PATH: "
-                "TypeError('len() takes exactly one argument (0 given)'); "
+                "TypeError('type object argument after * must be an iterable, not int'); "
```

Comment:
@navinkarkera This test case seems to be failing in my local env with the following error. Please take a look at this.

```
>       assert [str(w.message) for w in caught_warnings] == warning_messages
E       assert ['Failed to l... be inactive'] == ['Failed to l... be inactive']
E         At index 0 diff: "Failed to load <class 'dict'> from setting EB_LOAD_PATH: TypeError('dict() argument after * must be an iterable, not int'); component will be inactive" != "Failed to load <class 'dict'> from setting EB_LOAD_PATH: TypeError('type object argument after * must be an iterable, not int'); component will be inactive"
E         Use -v to get more diff

```

Reply:
@kaustavb12 Weird, it is working for me. Can you make sure your python version is 3.8 and the requirements are updated. (remove codecov from the requirements/dev.txt file manually).

Reply:
@navinkarkera Actually I was using python version 3.10. Switching to 3.8, the tests ran successfully as it is.